tv_apps/smart-home/js/card_picker.js

```diff
@@ -233,8 +233,8 @@
     },
 
     _scrollTo: function(elem) {
-      var scrollY = elem.offsetTop -
-              (this.gridView.offsetHeight - elem.offsetHeight) / 2;
+      var scrollY = elem.parentNode.offsetTop -
```

Comment:
I'd prefer calling `_scrollTo(elem.parentNode)` instead in order to make this function name more meaningful.